Dockerfile

```diff
@@ -1,26 +1,17 @@
-FROM debian:sid
+FROM openjdk:8-jre-alpine
 
 MAINTAINER Christian Kuntzsch <christian@kuntzsch.me>
 
-RUN \
-    apt-get update && \
```

Comment:
If you run `apt-get update` you should remove the package information after that to keep the image as small as possible...